hw/top_earlgrey/ip/ast/lint/ast.waiver

```diff
@@ -25,7 +25,11 @@ waive -rules IFDEF_CODE -location {aon_osc.sv io_osc.sv sys_osc.sv usb_osc.sv} \
       -comment {This ifdef statement is fine as it is part of the FPGA/Verilator clock bypass mechanism.}
```

Comment:
i still wonder if long term if it's just better to blackbox ast entirely.

Reply:
Yeah there are plenty of things that can just be waived since it is only a model. 
If we don't wan't to fully black-box we could blanket-waive all CLOCK/RESET/IFDEF related things inside, though.
That would simplify the waivers considerably.

Reply:
ah, i did not read that right. sounds good.

Reply:
since we already went through this process..maybe no need to redo.
I think it'd be nice in the future to black box stuff... but we probably would need to confirm Nuvoton which ones are okay to blackbox.

Reply:
yeah somewhat simiar to how we deal with CDC where we add an abstraction for the module.
that way we don't have to sift through all of this (and it always changes a lot when the module is re-delivered...).